src/selenium/schedulePage.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const SchedulePage = Object.create(BasePage);
+const datesId = ['2017-03-17', '2017-03-18', '2017-03-19'];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement. (one-var)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)